src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1195,6 +1213,7 @@ static bool processContextAttributeVector (ContextElement*
                                       " - entity: [" + eP->toString() + "]" +
                                       " - offending attribute: " + targetAttr->toString());
                 *why = ATTRIBUTE_NOT_FOUND;
+                LM_W(("Bad Input (attribute not found)"));
```

Comment:
Added a warning per 'offending attribute'.
Got tired of entering this function to understand the error when getting a 'offending attribute'.